src/HealthChecks.EventStore.gRPC/HealthChecks.EventStore.gRPC.csproj

```diff
@@ -10,6 +10,7 @@
   <ItemGroup>
     <PackageReference Include="EventStore.Client.Grpc.Streams" Version="22.0.0" />
     <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks" Version="7.0.1" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
```

Comment:
Let's do a small hack taking this new dep under condition of net6 and adding net7.0 to the list of target frameworks. So for net7 there will be no explicit dependency from Microsoft.Bcl.AsyncInterfaces.

Reply:
Sounds good, done 👍